lib/absinthe/schema/notation/sdl_render.ex

```diff
@@ -306,12 +306,54 @@ defmodule Absinthe.Schema.Notation.SDL.Render do
     )
   end
 
-  defp default(nil) do
+  defp default(%Blueprint.Schema.FieldDefinition{default_value: nil}) do
     empty()
   end
 
-  defp default(default_value) do
-    concat([" = ", render_value(default_value)])
+  defp default(%Blueprint.Schema.FieldDefinition{default_value: {:ref, _, _}}) do
+    empty()
+  end
+
+  defp default(%Blueprint.Schema.FieldDefinition{default_value: default_value})
+       when is_atom(default_value) or is_binary(default_value) or is_number(default_value) or
+              is_boolean(default_value) or is_list(default_value) or is_map(default_value) do
+    try do
+      blueprint_value = Blueprint.Input.parse(default_value)
+      concat([" = ", render_value(blueprint_value)])
+    rescue
+      _ -> empty()
+    end
```

Comment:
What led to using `rescue` here? Ideally we don't use exceptions for control flow... This could swallow actual errors

Reply:
sorry, just missed cleaning these up while debugging, let me polish this up